### ISSUE TITLE
Cannot build from source on windows

### ISSUE THREAD

**Issue by kkriske** [oracle/graal#3604]
Windows build of master seems to fail on compilation of module-info.java files. I have built it in the exact same way before so it would be unlikely to be caused by local setup, since I didn't actively modify that.

**Steps to reproduce the issue**
Please include both build steps as well as run steps
1. `git clone https://github.com/oracle/graal.git`
2. cd to the `substratevm` directory
3. `mx "--java-home=C:\Program Files\Java\labsjdk-ce-11.0.12-jvmci-21.2-b05" build`

**Describe GraalVM and your environment:**
 - current master, which would be 455ea952b6f15c277020378055a10a6f5d8c0220
 - JDK major version: **11**
 - OS: **Windows**
 - Architecture: **AMD64**

**More details**
Add any other information about the problem here. Especially important are stack traces or log output. Feel free to link to gists or to screenshots if necessary.

<details>
<summary>full build log</summary>

```
C:\Users\KristofDhondt\IdeaProjects\graal-fork\substratevm>mx "--java-home=C:\Program Files\Java\labsjdk-ce-11.0.12-jvmci-21.2-b05" build
WARNING: symlinking not supported
JAVA_HOME: C:\Program Files\Java\labsjdk-ce-11.0.12-jvmci-21.2-b05
29 unsatisfied dependencies were removed from build (use -v to list them)
1 non-default dependencies were removed from build (use -v to list them, mx build --all to build them)
Compiling com.oracle.mxtool.compilerserver with javac(JDK 11)... [C:\Comsof\projects\mx\mxbuild\java\com.oracle.mxtool.compilerserver\bin\com\oracle\mxtool\compilerserver\CompilerDaemon.class does not exist]
Compiling org.graalvm.compiler.truffle.runtime.serviceprovider with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\src\org.graalvm.compiler.truffle.runtime.serviceprovider\bin\org\graalvm\compiler\truffle\runtime\serviceprovider\TruffleRuntimeServices.class does not exist]
Compiling org.graalvm.compiler.truffle.runtime.serviceprovider.jdk11 with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.truffle.runtime.serviceprovider updated]
Compiling org.graalvm.compiler.api.directives with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\src\org.graalvm.compiler.api.directives\bin\org\graalvm\compiler\api\directives\GraalDirectives.class does not exist]
Compiling org.graalvm.compiler.api.runtime with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\src\org.graalvm.compiler.api.runtime\bin\org\graalvm\compiler\api\runtime\GraalJVMCICompiler.class does not exist]
Compiling org.graalvm.compiler.api.test with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.api.runtime updated]
Compiling org.graalvm.compiler.processor with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\src\org.graalvm.compiler.processor\bin\org\graalvm\compiler\processor\SuppressFBWarnings.class does not exist]
Compiling org.graalvm.compiler.options.processor with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.processor updated]
Compiling org.graalvm.compiler.serviceprovider.processor with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.processor updated]
Compiling org.graalvm.compiler.nodeinfo.processor with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.processor updated]
Compiling org.graalvm.compiler.replacements.processor with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.processor updated]
Compiling org.graalvm.compiler.core.match.processor with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.processor updated]
Compiling org.graalvm.compiler.lir.processor with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.processor updated]
Archiving GRAAL_PROCESSOR... [dependency org.graalvm.compiler.processor updated]
Compiling org.graalvm.compiler.nodeinfo with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\src\org.graalvm.compiler.nodeinfo\bin\org\graalvm\compiler\nodeinfo\NodeInfo.class does not exist]
Compiling org.graalvm.collections with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\sdk\mxbuild\src\org.graalvm.collections\bin\org\graalvm\collections\EconomicSet.class does not exist]
Compiling org.graalvm.word with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\sdk\mxbuild\src\org.graalvm.word\bin\org\graalvm\word\ComparableWord.class does not exist]
Compiling org.graalvm.options with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\sdk\mxbuild\src\org.graalvm.options\bin\org\graalvm\options\OptionStability.class does not exist]
Compiling org.graalvm.nativeimage with javac-daemon(JDK 11)... [dependency org.graalvm.word updated]
Compiling org.graalvm.home with javac-daemon(JDK 11)... [dependency org.graalvm.nativeimage updated]
Compiling org.graalvm.polyglot with javac-daemon(JDK 11)... [dependency org.graalvm.collections updated]
Archiving GRAAL_SDK... [dependency org.graalvm.polyglot updated]
Building Java module org.graalvm.sdk (graal-sdk.jar) from GRAAL_SDK
Compiling org.graalvm.util with javac-daemon(JDK 11)... [dependency GRAAL_SDK updated]
Compiling org.graalvm.compiler.serviceprovider with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\src\org.graalvm.compiler.serviceprovider\bin\org\graalvm\compiler\serviceprovider\SpeculationEncodingAdapter.class does not exist]
Compiling org.graalvm.graphio with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\src\org.graalvm.graphio\bin\org\graalvm\graphio\DefaultGraphBlocks.class does not exist]
Compiling org.graalvm.compiler.options with javac-daemon(JDK 11)... [dependency GRAAL_SDK updated]
Compiling org.graalvm.compiler.debug with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.core.common with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.bytecode with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\src\org.graalvm.compiler.bytecode\bin\org\graalvm\compiler\bytecode\BridgeMethodUtils.class does not exist]
Compiling org.graalvm.compiler.graph with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.code with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.api.replacements with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\src\org.graalvm.compiler.api.replacements\bin\org\graalvm\compiler\api\replacements\Fold.class does not exist]
Compiling org.graalvm.compiler.asm with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.core.common updated]
Compiling org.graalvm.compiler.lir with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.nodes with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.word with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.phases with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.phases.common with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.virtual with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.loop.phases with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.core with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.runtime with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.core updated]
Compiling org.graalvm.compiler.test with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.debug updated]
Compiling org.graalvm.compiler.asm.test with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.api.test updated]
Compiling org.graalvm.compiler.asm.amd64 with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.asm.amd64.test with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.asm.test updated]
Compiling org.graalvm.compiler.truffle.common with javac-daemon(JDK 11)... [dependency org.graalvm.graphio updated]
Compiling com.oracle.truffle.dsl.processor with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\truffle\mxbuild\src\com.oracle.truffle.dsl.processor\bin\com\oracle\truffle\dsl\processor\library\ExportsParser.class does not exist]
Compiling com.oracle.truffle.dsl.processor.jdk9 with javac-daemon(JDK 11)... [dependency com.oracle.truffle.dsl.processor updated]
Compiling com.oracle.truffle.object.dsl.processor with javac-daemon(JDK 11)... [dependency com.oracle.truffle.dsl.processor updated]
Archiving TRUFFLE_DSL_PROCESSOR... [dependency com.oracle.truffle.dsl.processor.jdk9 updated]
Compiling com.oracle.truffle.api.jdk11 with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\truffle\mxbuild\src\com.oracle.truffle.api.jdk11\bin\com\oracle\truffle\api\impl\TruffleJDKServices.class does not exist]
Compiling com.oracle.truffle.api with javac-daemon(JDK 11)... [dependency GRAAL_SDK updated]
Compiling com.oracle.truffle.api.dsl with javac-daemon(JDK 11)... [dependency com.oracle.truffle.api updated]
Compiling com.oracle.truffle.api.utilities with javac-daemon(JDK 11)... [dependency com.oracle.truffle.api updated]
Compiling com.oracle.truffle.api.library with javac-daemon(JDK 11)... [dependency TRUFFLE_DSL_PROCESSOR updated]
Compiling com.oracle.truffle.api.profiles with javac-daemon(JDK 11)... [dependency com.oracle.truffle.api updated]
Compiling com.oracle.truffle.api.interop with javac-daemon(JDK 11)... [dependency TRUFFLE_DSL_PROCESSOR updated]
Compiling com.oracle.truffle.api.exception with javac-daemon(JDK 11)... [dependency TRUFFLE_DSL_PROCESSOR updated]
Compiling com.oracle.truffle.api.instrumentation with javac-daemon(JDK 11)... [dependency TRUFFLE_DSL_PROCESSOR updated]
Compiling com.oracle.truffle.polyglot with javac-daemon(JDK 11)... [dependency TRUFFLE_DSL_PROCESSOR updated]
Compiling com.oracle.truffle.api.debug with javac-daemon(JDK 11)... [dependency TRUFFLE_DSL_PROCESSOR updated]
Compiling com.oracle.truffle.api.object with javac-daemon(JDK 11)... [dependency TRUFFLE_DSL_PROCESSOR updated]
Compiling com.oracle.truffle.object with javac-daemon(JDK 11)... [dependency TRUFFLE_DSL_PROCESSOR updated]
Compiling com.oracle.truffle.api.object.dsl with javac-daemon(JDK 11)... [dependency com.oracle.truffle.api.object updated]
Compiling com.oracle.truffle.host with javac-daemon(JDK 11)... [dependency TRUFFLE_DSL_PROCESSOR updated]
Archiving TRUFFLE_API... [dependency com.oracle.truffle.api.jdk11 updated]
Building Java module org.graalvm.truffle (truffle-api-open.jar) from TRUFFLE_API
Building Java module org.graalvm.truffle (truffle-api.jar) from TRUFFLE_API
Compiling org.graalvm.compiler.truffle.options with javac-daemon(JDK 11)... [dependency TRUFFLE_DSL_PROCESSOR updated]
Compiling org.graalvm.compiler.java with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.replacements with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.truffle.compiler with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.libgraal with javac-daemon(JDK 11)... [dependency GRAAL_SDK updated]
Compiling org.graalvm.libgraal.jdk11 with javac-daemon(JDK 11)... [dependency org.graalvm.libgraal updated]
Compiling org.graalvm.libgraal.jni.annotation with javac-daemon(JDK 11)... [C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\src\org.graalvm.libgraal.jni.annotation\bin\org\graalvm\libgraal\jni\annotation\Utilities.class does not exist]
Compiling org.graalvm.libgraal.jni.processor with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.processor updated]
Compiling org.graalvm.compiler.truffle.common.hotspot.libgraal with javac-daemon(JDK 11)... [dependency org.graalvm.libgraal.jni.annotation updated]
Compiling org.graalvm.compiler.truffle.compiler.hotspot.libgraal.processor with javac-daemon(JDK 11)... [dependency org.graalvm.libgraal.jni.processor updated]
Compiling org.graalvm.compiler.hotspot.management.libgraal.annotation with javac-daemon(JDK 11)... [dependency org.graalvm.libgraal.jni.annotation updated]
Compiling org.graalvm.compiler.hotspot.management.libgraal.processor with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.hotspot.management.libgraal.annotation updated]
Archiving GRAAL_LIBGRAAL_PROCESSOR... [dependency org.graalvm.libgraal.jni.processor updated]
Compiling org.graalvm.nativebridge.jni with javac-daemon(JDK 11)... [dependency GRAAL_SDK updated]
Compiling org.graalvm.libgraal.jni with javac-daemon(JDK 11)... [dependency GRAAL_LIBGRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.options.jdk11 with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.options updated]
Compiling org.graalvm.compiler.serviceprovider.jdk11 with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.serviceprovider updated]
Compiling org.graalvm.compiler.printer with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.asm.aarch64 with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.asm updated]
Compiling org.graalvm.compiler.lir.aarch64 with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.core.aarch64 with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.replacements.aarch64 with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.lir.amd64 with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.replacements.amd64 with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.core.amd64 with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.hotspot with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.hotspot.aarch64 with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.hotspot.amd64 with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.hotspot.jdk11 with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.hotspot updated]
Compiling org.graalvm.compiler.truffle.compiler.amd64 with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.truffle.jfr with javac-daemon(JDK 11)... [dependency TRUFFLE_API updated]
Compiling org.graalvm.compiler.truffle.runtime with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.truffle.common.hotspot with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.truffle.runtime.hotspot with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.truffle.runtime updated]
Compiling org.graalvm.compiler.truffle.runtime.hotspot.jdk11 with javac-daemon(JDK 11)... [dependency org.graalvm.compiler.truffle.runtime.hotspot updated]
Compiling org.graalvm.compiler.truffle.compiler.hotspot with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.truffle.runtime.hotspot.java with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.truffle.runtime.hotspot.libgraal with javac-daemon(JDK 11)... [dependency TRUFFLE_DSL_PROCESSOR updated]
Compiling org.graalvm.compiler.truffle.compiler.hotspot.amd64 with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Compiling org.graalvm.compiler.truffle.compiler.hotspot.aarch64 with javac-daemon(JDK 11)... [dependency GRAAL_PROCESSOR updated]
Archiving GRAAL... [dependency org.graalvm.libgraal.jdk11 updated]
Building Java module jdk.internal.vm.compiler (graal.jar) from GRAAL
C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\dists\jdk11\graal.jar.files\module-info.java:119: error: package is empty or does not exist: org.graalvm.compiler.truffle.common.hotspot.libgraal
    exports org.graalvm.compiler.truffle.common.hotspot.libgraal to com.oracle.graal.graal_enterprise, com.oracle.svm.svm_enterprise, org.graalvm.nativeimage.builder, org.graalvm.nativeimage.llvm, org.graalvm.nativeimage.pointsto;
                                                       ^
C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\dists\jdk11\graal.jar.files\module-info.java:138: error: package is empty or does not exist: org.graalvm.compiler.truffle.runtime.hotspot.libgraal
    exports org.graalvm.compiler.truffle.runtime.hotspot.libgraal to com.oracle.graal.graal_enterprise, com.oracle.svm.svm_enterprise, org.graalvm.nativeimage.builder, org.graalvm.nativeimage.llvm, org.graalvm.nativeimage.pointsto;
                                                        ^
C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\dists\jdk11\graal.jar.files\module-info.java:139: error: package is empty or does not exist: org.graalvm.compiler.truffle.runtime.serviceprovider
    exports org.graalvm.compiler.truffle.runtime.serviceprovider to com.oracle.graal.graal_enterprise, com.oracle.svm.svm_enterprise, org.graalvm.nativeimage.builder, org.graalvm.nativeimage.llvm, org.graalvm.nativeimage.pointsto;
                                                ^
C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\dists\jdk11\graal.jar.files\module-info.java:145: error: package is empty or does not exist: org.graalvm.libgraal.jni
    exports org.graalvm.libgraal.jni to com.oracle.graal.graal_enterprise, com.oracle.svm.svm_enterprise, org.graalvm.nativeimage.builder, org.graalvm.nativeimage.llvm, org.graalvm.nativeimage.pointsto;
                                ^
C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\dists\jdk11\graal.jar.files\module-info.java:146: error: package is empty or does not exist: org.graalvm.libgraal.jni.annotation
    exports org.graalvm.libgraal.jni.annotation to com.oracle.graal.graal_enterprise, com.oracle.svm.svm_enterprise, org.graalvm.nativeimage.builder, org.graalvm.nativeimage.llvm, org.graalvm.nativeimage.pointsto;
                                    ^
C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\dists\jdk11\graal.jar.files\module-info.java:147: error: package is empty or does not exist: org.graalvm.nativebridge.jni
    exports org.graalvm.nativebridge.jni to com.oracle.graal.graal_enterprise, com.oracle.svm.svm_enterprise, org.graalvm.nativeimage.builder, org.graalvm.nativeimage.llvm, org.graalvm.nativeimage.pointsto;
                                    ^
C:\Users\KristofDhondt\IdeaProjects\graal-fork\compiler\mxbuild\dists\jdk11\graal.jar.files\module-info.java:174: error: package org.graalvm.compiler.truffle.runtime.hotspot.libgraal does not exist
    provides com.oracle.truffle.api.TruffleRuntimeAccess with org.graalvm.compiler.truffle.runtime.hotspot.java.HotSpotTruffleRuntimeAccess, org.graalvm.compiler.truffle.runtime.hotspot.libgraal.LibGraalTruffleRuntimeAccess;
                                                                                                                                                                                                  ^
7 errors
```

</details>

**Comment by rodrigar-mx** [comment]
Hi @kkriske. Let me take a closer look at it, and try it myself. Any questions or updates, I will let you know.

**Comment by rodrigar-mx** [comment]
@kkriske try running it with admin privileges in order to permit symlinks

**Comment by kkriske** [comment]
@rodrigar-mx So far that does seem to fix my issue.

**Comment by stooke** [comment]
I am having exactly the same issue, and I'm running as administrator.
I do see that I get a warning from mx "WARNING: symlinking not supported", but I don't remember if that's new.
(this warning means that Python is reporting symlinking is not supported).
I'll explore further tomorrow.

**Comment by kkriske** [comment]
I also still encountered the issue in elevated cmd with python 2, build with python 3 seems to work.
I do however remember specifically setting up pythyon 2 for the GraalVM build, so is the build pipeline updated to require python 3?
Would be nice to get an appropriate exception if that's the case.

**Comment by dougxc** [comment]
The way mx builds jar distributions was [recently updated](https://github.com/graalvm/mx/commit/fb36392d3c4d0168222dbe6d291b1c2825ae2a4d). The Window specific behavior wrt symlinks is described under https://github.com/graalvm/mx#java-distributions.

Everything should still work on python2 however so seems like there's a bug there. That said, mx support for python2 is in the process of being removed (/cc @sanzinger ) so it's best to start using python3 anyway.

**Comment by kkriske** [comment]
@dougxc The symlinks are older behaviour though right? I do remember seeing the symlink warning before (or maybe I'm starting to see things).
I have to run elevated and in python 3, running just python 3 doesn't change anything so it looks a bit twofold.

The docs you linked state:
> if symlinks cannot be created, files are copied instead.

So normally the build should still work when not running elevated.

I did find a random blogpost about python and symlinks stating:
> If it’s still not possible to create symbolic links, you may simply need to use Python ≥ 3.8.

and 
> Python os.symlink() on Windows generally requires Python ≥ 3.8

Which I guess is another thing I can try, since I'm still on python 3.7, but it doesn't make a lot of sense since running elevated does work.. Which version would be required?

In any case, I have a working flow again. @stooke please update us if this also resolves it for you.

**Comment by dougxc** [comment]
I found the problem and am pushing a fix: https://github.com/graalvm/mx/commit/a0b8fee48fa9f2d968c012e1c32c1e3ab25430a9

**Comment by stooke** [comment]
@dougxc thanks - that fixes it for me, with the original python 2.7
